src/store/tokenStore.js

```diff
@@ -80,6 +95,10 @@ const actions = {
 	updateTokenAndFileIdForToken(context, { newToken, newFileId }) {
 		context.commit('updateTokenAndFileIdForToken', { newToken, newFileId })
 	},
+
+	handleRoomJoined({ commit }, token) {
```

Comment:
why not then call it updateLastJoinedConversationToken ? then at least we have only one term to think about.

I like that the "lastJoinedConversationToken" is well-documented, so my brain can associate the dual-states to that, but having `handleRoomJoined' would mean yet another wording for the same thing. Hence my query for renaming it.

Reply:
addressed!